db/warehouse/migrate/20230429212740_adjust_remote_credential_data.rb

```diff
@@ -0,0 +1,28 @@
+class AdjustRemoteCredentialData < ActiveRecord::Migration[6.1]
+  # use prefixed slugs for remote credentials. Set the namespace
+  # on the relevant external ids
+  def up
+    mapping.each do |old_name, new_name, namespace|
```

Comment:
Perhaps this data migration isn't needed since I don't know that any data exists outside of test dbs

Reply:
No harm as far as I can tell.